components/collective-page/index.js

```diff
@@ -78,11 +89,14 @@ class CollectivePage extends Component {
 
   constructor(props) {
     super(props);
+    // @deprecated This will be useless once new navbar will be the default
     this.sectionsRefs = {}; // This will store a map of sectionName => sectionRef
+    this.categoriesRefs = {}; // This will store a map of category => ref
```

Comment:
Out of simplicity, I've used the "category" word alone when we're in the context of the navbar, because there's not much confusion in this context. But it's true that this one can be confusing, since it's the collective page. I'll update now